res/res_pjsip_sdp_rtp.c

```diff
@@ -2004,6 +2070,15 @@ static int create_outgoing_sdp_stream(struct ast_sip_session *session, struct as
 			}
 		}
 
+		/* if we are using text with rtp redundancy, we need to match the t140 payload inside the RED sdp */
+		if (ast_format_get_type(format) == AST_MEDIA_TYPE_TEXT && strncasecmp(ast_format_get_name(format), "RED", 3) == 0) {
+			if (*((int *)ast_format_attribute_get(format, "red_payload")) != t140_payload) {
```

Comment:
What would happen here if the `res_format_attr_red` module was not loaded? If `ast_format_attribute_get(...)` returns `NULL` in that case we're in trouble.

Reply:
Ok, lets add another check here.